src/lib/components/LivestreamContainer/index.js

```diff
@@ -69,14 +70,29 @@ class LivestreamContainer extends BaseStateElement {
   /**
    * @param {!Object<string, *>} state
    */
-  onStateChanged({activeEventDay}) {
-    if (!activeEventDay) {
-      return;
-    }
+  onStateChanged({activeEventDay, isSignedIn}) {
+    const {videoId, isChatActive} = activeEventDay || {
+      videoId: null,
+      isChatActive: false,
+    };
 
-    const {videoId, isChatActive} = activeEventDay;
+    const updateFrame = this.videoId !== videoId;
```

Comment:
```suggestion
    const shouldUpdateIFrames = this.videoId !== videoId;
```

Bikeshedding (sorry) but I like using `is`, `should`, or `has` to quickly signify if something is going to be a boolean vs a function.

Reply:
Moot since I've removed this boolean